backend/src/test/java/trainingjournal/backend/controller/PlanControllerTest.java

```diff
@@ -0,0 +1,61 @@
+package trainingjournal.backend.controller;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.web.servlet.MockMvc;
+import trainingjournal.backend.repository.ExerciseRepository;
+import trainingjournal.backend.repository.PlanRepository;
+import trainingjournal.backend.service.PlanService;
+
+import static org.springframework.security.test.web.servlet.request.SecurityMockMvcRequestPostProcessors.csrf;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@AutoConfigureMockMvc
+class PlanControllerTest {
+
+    @Autowired
+    MockMvc mockMvc;
+
+    @Autowired
+    ExerciseRepository exerciseRepository;
+    @Autowired
+    PlanRepository planRepository;
+
+    @WithMockUser(username="StandardUser")
+    @Test
+    void test_getAllExercises() throws Exception{
```

Comment:
Bei den Namen gerne etwas detailierter sein - wir sehen ja das es ein Test ist, aber was ist das erwartete verhalten in diesem Fall?

Probier doch mal so etwas aus wie `getAllExercises_shouldReturnEmptyList_whenDbIsEmpty` oder so etwas

Reply:
Gilt für alle Tests im Projekt